packages/flutter_tools/lib/src/ios/xcodeproj.dart

```diff
@@ -28,21 +25,30 @@ String flutterFrameworkDir(BuildMode mode) {
   return fs.path.normalize(fs.path.dirname(artifacts.getArtifactPath(Artifact.flutterFramework, TargetPlatform.ios, mode)));
 }
 
-String _generatedXcodePropertiesPath(String projectPath) {
-  return fs.path.join(projectPath, 'ios', 'Flutter', 'Generated.xcconfig');
+String _generatedXcodePropertiesPath(String projectPath, FlutterManifest manifest) {
+  if (manifest.isModule) {
+    return fs.path.join(projectPath, '.ios', 'Generated.xcconfig');
+  } else {
+    return fs.path.join(projectPath, 'ios', 'Flutter', 'Generated.xcconfig');
```

Comment:
Not sure why we want to avoid the `Flutter` folder here? We do have an `.android/Flutter` sub project.